src/OpenTelemetry.Contrib.Instrumentation.EventCounters/SystemRuntime/SystemRuntimeOptionsExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// <copyright file="SystemRuntimeOptionsExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using OpenTelemetry.Contrib.Instrumentation.EventCounters;
+using OpenTelemetry.Contrib.Instrumentation.EventCounters.SystemRuntime;
+
+namespace OpenTelemetry.Metrics
+{
+    /// <summary>
+    /// Extension methods for the event counters options.
+    /// </summary>
+    public static class SystemRuntimeOptionsExtensions
```

Comment:
maybe good to keep the initial scope smaller. All this (along with grpc/aspnetcore) can be added by users without these as well. If there is a need, we can always add new things.

Reply:
So should we remove these predefined counters or should we move those to a separate project each?

Reply:
remove now. The readme.md can show how users can enable them. (all they need is the name of eventsource,counter, whether to treat it as gauge/counter).
Once this PR is merged, we can discuss the best options on how to support the well known counters - either as part of this project itself, or a separate project (nugets) etc.